src/Form/Type/BasePickerType.php

```diff
@@ -72,9 +72,12 @@ public function finishView(FormView $view, FormInterface $form, array $options)
         if (isset($options['date_format']) && is_string($options['date_format'])) {
             $format = $options['date_format'];
         } elseif (is_int($format)) {
-            $timeFormat = ($options['dp_pick_time']) ? DateTimeType::DEFAULT_TIME_FORMAT : \IntlDateFormatter::NONE;
+            $timeFormat = \IntlDateFormatter::NONE;
+            if ($options['dp_pick_time']) {
```

Comment:
shouldn't there be a new line before if?

Reply:
strange that flintCI didn't report this?

Reply:
I don't think there is any CS rule about this…